app/services/doc_auth/response.rb

```diff
@@ -14,7 +14,9 @@ def initialize(
       extra: {},
       pii_from_doc: {},
       attention_with_barcode: false,
-      doc_type_supported: true
+      doc_type_supported: true,
+      # This is a stub to get data paths in place. Replace this as soon as possible.
+      selfie_check_performed: IdentityConfig.store.doc_auth_selfie_capture_enabled
```

Comment:
We need to check whether the sp requested a selfie (which also checks the config value). If the session isn't available here we'll need to pass in the value.
```suggestion
      selfie_check_performed: decorated_sp_session.selfie_required?
```

Reply:
I agree that `#selfie_required?` would be better, but since this is stub code anyway, I'm really reluctant to pass in the decorated session. The other alternative is to remove the default value for `selfie_check_performed` and pass in a value everywhere it's called, and I like that even less for stub code.